cmd/rpc.go

```diff
@@ -175,3 +188,44 @@ func startNativeSpaceBridgeRpcServer(ctx context.Context, wg *sync.WaitGroup) {
 	server := rpc.MustNewNativeSpaceBridgeServer(&config)
 	go server.MustServeGraceful(ctx, wg, config.Endpoint, rpcutil.ProtocolHttp)
 }
+
+type ForwardHandler struct{}
+
+func (h *ForwardHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+
+	valid, err := whitelist.IsIPValid(req)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	if valid == false {
+		return //FIXME
+	}
+
+	URL := "http://localhost" + viper.GetString("debugnode.endpoint")
```

Comment:
For cfx and eth the gateway allows specifying a group (1 or more nodes) and uses `Route` to select between these. I think we might need this capability here too, in case we need to scale up the debug RPCs. https://github.com/scroll-tech/rpc-gateway/blob/staging/node/client.go#L62

Reply:
added